app/src/main/res/layout/item_album.xml

```diff
@@ -8,49 +9,108 @@
 
     </data>
 
-    <androidx.cardview.widget.CardView
+
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/cardview_background"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="0dp"
-        android:layout_marginLeft="3dp"
-        android:layout_marginRight="3dp"
-        android:layout_marginTop="1dp"
-        app:cardElevation="4dp"
-        app:cardUseCompatPadding="true">
-
-        <ImageView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:src="@{viewModel.url}"
-            android:contentDescription="@{viewModel.title}"
-            android:onClick="@{viewModel::onClickDo}"
-            app:heightRatio="@{0.66}"
-            android:cropToPadding="false"
-            android:scaleType="centerCrop"
-            />
-
-        <LinearLayout
+        android:layout_height="150dp"
+        android:layout_marginTop="4dp"
+        >
+
+        <com.vipulasri.ticketview.TicketView
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="bottom"
-            android:background="@color/white_semi_transparent"
-            android:orientation="vertical"
-            android:padding="16dp">
+            android:layout_height="match_parent"
+            android:id="@+id/ticketView"
+            app:ticketOrientation="vertical"
+            app:ticketScallopRadius="10dp"
+            app:ticketCornerType="rounded"
+            app:ticketCornerRadius="12dp"
+            app:ticketScallopPositionPercent="40"
+            app:ticketElevation="14dp"/>
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/first_card"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            app:cardCornerRadius="12dp"
+            app:cardElevation="0dp"
+            android:layout_marginEnd="13dp"
+            android:layout_marginStart="13dp"
+            android:layout_marginTop="9dp"
+            android:layout_marginBottom="15dp"
+            app:cardBackgroundColor="@color/colorAccent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/guideline"
+            app:layout_constraintBottom_toBottomOf="parent"
+            >
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:src="@{viewModel.url}"
+                android:contentDescription="@{viewModel.title}"
+                android:onClick="@{viewModel::onClickDo}"
+                android:scaleType="fitXY"
```

Comment:
this one looks suspicious for the aspect ratio... Do you want to try it with CENTER_CROP?

Reply:
Here is look with CENTER_CROP
![piwigo3](https://user-images.githubusercontent.com/56266180/69579271-793fe200-0fd2-11ea-9d7a-de47f8c97929.PNG)